dbms/src/Flash/Planner/PhysicalPlan.cpp

```diff
@@ -53,6 +52,16 @@ bool pushDownSelection(const PhysicalPlanNodePtr & plan, const String & executor
     }
     return false;
 }
+
+void fillOrderForListBasedExecutors(DAGContext & dag_context, const PhysicalPlanNodePtr & root_node)
+{
+    auto & list_based_executors_order = dag_context.list_based_executors_order;
+    PhysicalPlanVisitor::visitPostOrder(root_node, [&](const PhysicalPlanNodePtr & plan) {
+        assert(plan);
+        if (plan->isRecordProfileStreams())
```

Comment:
likely？

Reply:
But `FinalProjection.isRecordProfileStreams() == false`, so I think it's better not to add likely